### PR TITLE
Update hiring page with details for senior developer and product mana…

### PR DESCRIPTION
**Pull request by BenJam**
as titled

- adds two roles (senior product manager and developer)
- reorganises page to account for multiple roles hiring at one point
- makes some other text changes

### CODE REVIEW

File: components/static-pages/hiring.md [hunk 1]
```diff
@@ -6,24 +6,36 @@
 <strong>All position are currently filled, check back later! You can still send a spontaneous application if you wish.</strong>
 -->
 
-<em>September 28th 2021</em>. We have an immediate need and are for a looking full stack <strong>free lance</strong> engineer who’s excited about open source, making communities sustainable, and transparent finances. The mission would be initially on a <strong>3 months</strong> contract. This could be later extended based on fit and availability.
+<em>January 28th 2021</em>. We are currently hiring for the following roles:
```
Comment (znarf) [inline components/static-pages/hiring.md:9]:
```suggestion
<em>January 12th 2022</em>. We are currently hiring for the following roles:
```

File: components/static-pages/hiring.md [hunk 2]
```diff
@@ -6,24 +6,36 @@
...
+<h2 id='developer'>Senior, 'Full Stack' (Node) Developer</h2>
+
+We are looking for a senior full-stack engineer who’s excited about open source, making communities sustainable, and transparent finances.
+
+### We think you’d be great at this if you:
 
 - Are comfortable in a remote environment that is at the same time highly collaborative (we [apply](https://github.com/opencollective/opencollective/issues/2267) a [seeking feedback process](https://reinventingorganizationswiki.com/theory/decision-making/)) and autonomous (you can run a feature end to end on your own!)
 
-- Are based in Europe:
+- Are based within 4 hours of Central European Time:
```
Comment (znarf) [inline components/static-pages/hiring.md:36]: This excludes US east coast?

File: components/static-pages/hiring.md [hunk 3]
```diff
@@ -6,24 +6,36 @@
 <strong>All position are currently filled, check back later! You can still send a spontaneous application if you wish.</strong>
 -->
 
-<em>September 28th 2021</em>. We have an immediate need and are for a looking full stack <strong>free lance</strong> engineer who’s excited about open source, making communities sustainable, and transparent finances. The mission would be initially on a <strong>3 months</strong> contract. This could be later extended based on fit and availability.
+<em>January 28th 2021</em>. We are currently hiring for the following roles:
 
-## First, about us
+<ul>
+  <li><a href='#developer'>Senior, 'Full Stack' (Node) Developer</a></li>
```
Comment (znarf) [inline components/static-pages/hiring.md:12]:
```suggestion
  <li><a href='#developer'>Senior, 'Full Stack' Software Engineer</a></li>
```

We usually don't use the word "Developer".

File: components/static-pages/hiring.md (hunk 2 above)
Comment (BenJam) [reply components/static-pages/hiring.md:36]:
```suggestion
- Are based within 5 hours of Central European Time:
```

File: components/static-pages/hiring.md [hunk 4]
```diff
@@ -6,24 +6,36 @@
...
+Watch this 2 minute explainer video about what we do and why we are passionate about it:
 
 <iframe src="https://www.youtube.com/embed/IBU5fSILAe8?wmode=opaque&amp;enablejsapi=1" scrolling="no" width="854" height="480" frameborder="0"></iframe>
 
-## We think you’d be great at this if you:
+Like the movements that we support, we value openness and transparency. We strive to create inclusive, respectful and sustainable communities and we're 100% Open Source. If you want to know more about us, you can already join our [Slack](https://slack.opencollective.com) and check out the [Open Collective Way](https://docs.opencollective.com/help/about/the-open-collective-way) to learn more about how we operate and what we stand for.
+
+We are a fully remote, international team - with team members in New Zealand, Mexico, India, Brazil, Canada, the United Kingdom, Spain, Germany and France. We are an equal opportunities employer and welcome applications from anyone with the right mix of experience and skills.
+
+<h2 id='developer'>Senior, 'Full Stack' (Node) Developer</h2>
```
Comment (znarf) [inline components/static-pages/hiring.md:28]:
```suggestion
<h2 id='software-engineer'>Senior, 'Full Stack' Software Engineer</h2>
```

File: components/static-pages/hiring.md [hunk 5]
```diff
@@ -6,24 +6,36 @@
 <strong>All position are currently filled, check back later! You can still send a spontaneous application if you wish.</strong>
 -->
 
-<em>September 28th 2021</em>. We have an immediate need and are for a looking full stack <strong>free lance</strong> engineer who’s excited about open source, making communities sustainable, and transparent finances. The mission would be initially on a <strong>3 months</strong> contract. This could be later extended based on fit and availability.
+<em>January 12th 2022</em>. We are currently hiring for the following roles:
 
-## First, about us
+<ul>
+  <li><a href='#developer'>Senior, 'Full Stack' Software Engineer</a></li>
```
Comment (znarf) [inline components/static-pages/hiring.md:12]:
```suggestion
  <li><a href='#software-engineer'>Senior, 'Full Stack' Software Engineer</a></li>
```

File: components/static-pages/hiring.md [hunk 6]
```diff
@@ -6,24 +6,36 @@
...
+<h2 id='developer'>Senior, 'Full Stack' (Node) Developer</h2>
+
+We are looking for a senior full-stack engineer who’s excited about open source, making communities sustainable, and transparent finances.
+
+### We think you’d be great at this if you:
 
 - Are comfortable in a remote environment that is at the same time highly collaborative (we [apply](https://github.com/opencollective/opencollective/issues/2267) a [seeking feedback process](https://reinventingorganizationswiki.com/theory/decision-making/)) and autonomous (you can run a feature end to end on your own!)
 
-- Are based in Europe:
+- Are based within 5 hours of Central European Time:
```
Comment (znarf) [inline components/static-pages/hiring.md:36]: 6 hours between CET and EST.

File: components/static-pages/hiring.md [hunk 7]
```diff
@@ -32,20 +44,39 @@ Watch this 2 minute explainer video about what we do and why we are passionate a
...
-Like the movements that we support, we value openness and transparency. We strive to create inclusive, respectful and sustainable communities and we're 100% Open Source. If you want to know more about us, you can already join our [Slack](https://slack.opencollective.com) and check out the [Open Collective Way](https://docs.opencollective.com/help/about/the-open-collective-way) to learn more about how we operate and what we stand for.
+<h2 id='product-manager'>Senior Product Manager</h2>
+
+We are looking for a senior product manager who is excited about creating a world in which it is as rewarding and financially secure to build and maintain software for the commons as it is for the corporation.
+
+### We think you’d be great at this if you:
+
+- Are comfortable in a remote environment that is at the same time highly collaborative (we [apply](https://github.com/opencollective/opencollective/issues/2267) a [seeking feedback process](https://reinventingorganizationswiki.com/theory/decision-making/)) and autonomous (you will own complete features and products!)
+
+- Are based within 4 hours of Central European time:
```
Comment (znarf) [inline components/static-pages/hiring.md:57]: Same timezone than the other?

File: components/static-pages/hiring.md [hunk 8]
```diff
@@ -6,24 +6,36 @@
...
+<h2 id='developer'>Senior, 'Full Stack' (Node) Developer</h2>
+
+We are looking for a senior full-stack engineer who’s excited about open source, making communities sustainable, and transparent finances.
+
+### We think you’d be great at this if you:
 
 - Are comfortable in a remote environment that is at the same time highly collaborative (we [apply](https://github.com/opencollective/opencollective/issues/2267) a [seeking feedback process](https://reinventingorganizationswiki.com/theory/decision-making/)) and autonomous (you can run a feature end to end on your own!)
 
-- Are based in Europe:
+- Are based within 5 hours of Central European Time:
```
Comment (znarf) [inline components/static-pages/hiring.md:36]:
```suggestion
- Are based within 6 hours of Central European Time:
```

File: components/static-pages/hiring.md (hunk 7 above)
Comment (znarf) [inline components/static-pages/hiring.md:57]:
```suggestion
- Are based within 6 hours of Central European time:
```

**Pull request merged**